### ISSUE TITLE
Which optimizer to use while fine-tuning ara-t5 base mode ?

### ISSUE THREAD

**Issue by mtala3t** [UBC-NLP/araT5#7]
Hello,

Please help , which optimizer should i use while fine-tuning your model ? and with which learning rate ?

Thanks.
